### DISCUSSION TITLE
Support for Sourcehut

### DISCUSSION THREAD

**Discussion by JamieMagee** [renovatebot/renovate#24884] [Suggest an Idea]
### What would you like Renovate to be able to do?

Add support for the [Sourcehut](https://sourcehut.org/) platform.

### If you have any ideas on how this should be implemented, please tell us here.

As it stands now, I am not sure Renovate would be a good fit as Sourcehut expects patches to be submitted via `git send-mail`[^1]. There is an open issue to add a UI for contributing code[^2], but it's unclear when this will be implemented.

I am mostly creating this issue to gather feedback, and allow people to :+1: it if it's something they think we should support.

[^1]: https://man.sr.ht/git.sr.ht/send-email.md
[^2]: https://todo.sr.ht/~sircmpwn/lists.sr.ht/111

### Is this a feature you are interested in implementing yourself?

No

**Comment by jthvai** [comment]
I'm not sure whether this would help much on the implementation front, but Sourcehut does seem to have a web UI for sending patches from a repo cloned to one's account[^1][^2]. This means that, at least, renovate wouldn't need to be hooked up to its own SMTP server.

[^1]: https://spacepub.space/w/ad258d23-0ac6-488c-83fc-2bacf578de3a 
[^2]: https://man.sr.ht/git.sr.ht/#sending-patches-upstream

**Comment by theoparis** [comment]
👍🏼  This would also mean supporting cgit and other methods of patch-based git hosting.
